fastlane_core/lib/fastlane_core/itunes_transporter.rb

```diff
@@ -180,9 +180,132 @@ def additional_upload_parameters
     end
   end
 
+  # Generates commands and executes the altool.
+  class AltoolTransporterExecutor < TransporterExecutor
+    ERROR_REGEX = /.*\*\*\*\sError:\s+(.+)/
+
+    private_constant :ERROR_REGEX
+
+    def execute(command, hide_output)
+      if Helper.test?
+        yield(nil) if block_given?
+        return command
+      end
+
+      @errors = []
+      @warnings = []
+      @all_lines = []
+
+      if hide_output
+        # Show a one time message instead
+        UI.success("Waiting for App Store Connect transporter to be finished.")
+        UI.success("Application Loader progress... this might take a few minutes...")
+      end
+
+      begin
+        exit_status = FastlaneCore::FastlanePty.spawn(command) do |command_stdout, command_stdin, pid|
+          begin
+            command_stdout.each do |line|
+              @all_lines << line
+              parse_line(line, hide_output) # this is where the parsing happens
+            end
+          end
+        end
+      rescue => ex
+        # FastlanePty adds exit_status on to StandardError so every error will have a status code
+        exit_status = ex.exit_status
+        @errors << ex.to_s
+      end
+
+      unless exit_status.zero?
+        @errors << "The call to the altool completed with a non-zero exit status: #{exit_status}. This indicates a failure."
+      end
+
+      if @warnings.count > 0
+        UI.important(@warnings.join("\n"))
+      end
+
+      if @errors.count > 0 && @all_lines.count > 0
+        # Print out the last 18 lines, this is key for non-verbose mode
```

Comment:
I found good way so please wait a moment

Reply:
Ah, sorry I think this line should be leave it as is, because the main purpose is providing altool's command output itself and regex is already done at here

https://github.com/fastlane/fastlane/blob/0f2a3d4e596d9b8d344a5b68270ab03d3daba0ab/fastlane_core/lib/fastlane_core/itunes_transporter.rb#L282-L283

Reply:
```
2022-09-09 16:12:21.904 Has version info conflict: YES, Previous version info: 15
2022-09-09 16:12:21.905 *** Error: Error uploading '***.ipa'.
2022-09-09 16:12:21.905 *** Error: The provided entity includes an attribute with a value that has already been used The bundle version must be higher than the previously uploaded version: ‘15’. (ID: ***) (-19232)
 {
    NSLocalizedDescription = "The provided entity includes an attribute with a value that has already been used";
    NSLocalizedFailureReason = "The bundle version must be higher than the previously uploaded version: ***";
    "original_server_error" =     {
        code = "ENTITY_ERROR.ATTRIBUTE.INVALID.DUPLICATE";
        detail = "The bundle version must be higher than the previously uploaded version.";
        id = "***";
        meta =         {
            previousBundleVersion = 15;
        };
        source =         {
            pointer = "/data/attributes/cfBundleVersion";
        };
        status = 409;
        title = "The provided entity includes an attribute with a value that has already been used";
    };
    previousBundleVersion = 15;
}
```

This is a sample and I found I may can improve around error, let me fix it